app/app.c

```diff
@@ -589,6 +589,23 @@ static void DualwatchAlternate(void)
 	#endif
 }
 
+///UNUSED	#define B0			(1<<0)
+#define FSKRX_SYNC				(1<<1)
+#define SQL_LOST				(1<<2)
+#define SQL_FOUND				(1<<3)
+#define VOX_LOST				(1<<4)
+#define VOX_FOUND				(1<<5)
+#define CTCSS_LOST				(1<<6)
+#define CTCSS_FOUND				(1<<7)
+#define CDCSS_LOST				(1<<8)
+#define CDCSS_FOUND				(1<<9)
+#define CSSTAIL_FOUND				(1<<10)
+#define DMTF5_TONE_FOUND			(1<<11)
+#define FSKFIFO_ALMOST_FULL			(1<<12) // pls define almost ?
+///UNUSED_ATM	#define FSK_RX_END		(1<<13)
+///UNUSED_ATM	#define FXKFIFO_ALMOST_EMPTY	(1<<14)
+///UNUSED_ATM	#define FSK_TX_FINISHED		(1<<15)
```

Comment:
The original name was fskTxFinied so the define could be FSK_TX_FINIED or FSK_TX_FINISHED (does 'finied' mean the same as 'finished'?)